.golangci.yml

```diff
@@ -87,6 +87,7 @@ issues:
         - dupl
         - funlen
         - gomnd
+        - errcheck
```

Comment:
We shouldn't exclude errcheck even for test files. We need to validate the returned error.

Reply:
@rolsonquadras fixed